Documentation/gettingstarted/aws.rst

```diff
@@ -0,0 +1,303 @@
+status of ``kube-dns`` you can run the following command:
+::
+
+    $ kubectl get deployment kube-dns -n kube-system
+    NAME       DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+    kube-dns   1         1         1            1           13h
+
+Where at least one pod should be available.
+
+
+Step 2: Create a sample policy
+================================
+
+Deploy a demo application:
+----------------------------
+
+In this case we're going to use a demo application that it is used in other
+guides. Example manifest will create three microservices applications:
+*deathstar*, *tiefighter*, and *xwing*. In this case, we are only going to use
+our *xwing* microservice to secure communications to existing AWS instances.
+
+.. parsed-literal::
+
+    $ kubectl create -f \ |SCM_WEB|\/examples/minikube/http-sw-app.yaml
+    service "deathstar" created
+    deployment "deathstar" created
+    deployment "tiefighter" created
+    deployment "xwing" created
+
+
+Kubernetes will deploy the pods and service in the background.  Running
+``kubectl get pods,svc`` will inform you about the progress of the operation.
+Each pod will go through several states until it reaches ``Running`` at which
+point the pod is ready.
+
+::
+
+    $ kubectl get pods,svc
+    NAME                             READY     STATUS    RESTARTS   AGE
+    po/deathstar-76995f4687-2mxb2    1/1       Running   0          1m
+    po/deathstar-76995f4687-xbgnl    1/1       Running   0          1m
+    po/tiefighter                    1/1       Running   0          1m
+    po/xwing                         1/1       Running   0          1m
+
+    NAME             TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)   AGE
+    svc/deathstar    ClusterIP   10.109.254.198   <none>        80/TCP    3h
+    svc/kubernetes   ClusterIP   10.96.0.1        <none>        443/TCP   3h
+
+Policy Language:
+-----------------
+
+To enforce security to AWS instances, the **ToGroups** rule will be used. This
+rule allows traffic to any cloud provider, in this example, AWS. The example
+policy will be:
+
+.. code-block:: yaml
+
+    ---
+    kind: CiliumNetworkPolicy
+    apiVersion: cilium.io/v2
+    metadata:
+      name: to-groups-sample
+      namespace: default
+    spec:
+      endpointSelector:
+        matchLabels:
+          org: alliance
+          class: xwing
+      egress:
+      - toPorts:
+        - ports:
+          - port: '80'
+            protocol: TCP
+        toGroups:
+        - aws:
+            securityGroupsIds:
+            - 'sg-0fef46100a88d03c3'
+
+This policy allows traffic from pod *xwing* to any AWS instance that is on
+the Security group with ID ``sg-0fef46100a88d03c3``. Also the rule can be by
+instance labels, security groups names or region.
+
+
+Validate that derived policy is in place
+----------------------------------------
+
+Every time that a new policy with ToGroups rules is added a new derivative
```

Comment:
This is a run-on sentence. I'd also change the 
"...is added, an equivalent policy (also called "derivative policy"), will be created. This policy will contain the set of CIDRs that correspond to the specification in ToGroups, e.g., the IPs of all instances that are part of a specified security group. "

Maybe specify that the policy is updated 'periodically' vs. 'every five minutes' - I'm worried that if this resync period is updated in the future that this documentation will fall out of sync with the codebase. This is nit-picky though :-)